### ISSUE TITLE
'chunk loop' loop is adding a 'row_num' property

### ISSUE THREAD

**Issue by mojosef** [laravel/framework#20814]
- Laravel Version: 5.4.32
- PHP Version: 7.1

### Description:
If you are on the last iteration of a 'chunk loop' loop, a 'row_num' property is added. If you then go to update and save that model, it fails as the query attempts to update a column 'row_num' on the model.

### Steps To Reproduce:
`\App\SomeModel::chunk(200, function($things){
    foreach ($things as $key => $thing) {
        $thing->foo = 'bar';
        $thing->save();
    }
});
Column not found: 1054 Unknown column 'row_num' in 'field list' (SQL: update some_model set foo = 'bar' updated_at = 2017-08-29 13:14:04, row_num = 201 where id = 201)`

**Comment by mnabialek** [comment]
What engine do you use? Are you ussing `sqlsrv`?

**Comment by mnabialek** [comment]
Yes, I think that's the reason, but I don't use that driver so cannot fix this easily.

**Comment by xorock** [comment]
I have the same problem. Temporary solution is to add 'row_num' to list of hidden fields in Your model.

```php
protected $hidden = [
        'row_num',
    ];
```
